pages/index.mdx

```diff
@@ -4,6 +4,11 @@ You are invited to join a two-day workshop on the various designs, applications,
 
 It will feature invited talks, tutorials and hackathon, student research presentations and more!
 
+## Important Dates
+- Submission deadline: Apr 30
```

Comment:
Specify which submission it is?

Reply:
The student submission in the call. I think we don't have other submissions for this workshop.

Reply:
There is a call for presentation in the google doc?